### PR TITLE
[15.0][MIG] partner_contact_lang: Migration to 15.0

### PR DESCRIPTION
**Pull request by pegonzalezspesol**
Work done:
- Changed version
- Adapted test class SavePointCase to run successfully tests in v15

### CODE REVIEW

File: partner_contact_lang/tests/test_partner_contact_lang.py [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 # Copyright 2018 Tecnativa - Cristina Martín
+# Copyright 2021 Pesol - Pedro Evaristo Gonzalez Sanchez
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
-from odoo.tests import common
+from odoo.addons.base.tests.common import SavepointCaseWithUserDemo
 
 
-class TestPartnerContactLang(common.SavepointCase):
+class TestPartnerContactLang(SavepointCaseWithUserDemo):
```
Comment (pedrobaeza) [inline partner_contact_lang/tests/test_partner_contact_lang.py]: Use `common.TransactionCase` as stated in the migration guide: https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-15.0

**Review by emagdalenaC2i — APPROVED** [review]
LGTM

**Review by dhx — APPROVED** [review]
Functional test looks good

**Pull request merged**